src/Symfony/Component/Cache/Adapter/PdoAdapterTrait.php

```diff
@@ -0,0 +1,325 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Cache\Adapter;
+
+use Symfony\Component\Cache\Exception\InvalidArgumentException;
+
+/**
+ * @internal
+ *
+ * @author Nicolas Grekas <p@tchwork.com>
```

Comment:
You sure? ;) This looks alot like copy-pasted from what I wrote for pdo session storage.